lib/compiler.js

```diff
@@ -29,6 +35,10 @@ function makeCallback(options) {
     if (error) {
       reject(error);
       return;
+    } else if (stats && stats.hasErrors() && !watch) {
+      process.exitCode = getErrorCount(stats);
```

Comment:
Please use `1` instead. Using the error count as the exit code when prevent _us_ from using meaningful non-zero exit codes.

Reply:
https://people.cs.pitt.edu/~alanjawi/cs449/code/shell/UnixSignals.htm

Reply:
Really looks strange

Reply:
resolved